Add information about icon alignment in Helios documentation (#1247)
hashicorp/design-system: website/docs/icons/usage-guidelines/partials/code/how-to-use.md

The size of the icon is determined by the size of the asset imported (each icon is exported in two sizes, _16_ and _24_). If you need a different size, use CSS to override its intrinsic size.

### Aligning icons

> By default, the `FlighIcon` component has a `inline-block` display value (this can be changed using the `@isInlineBlock` argument). This means that the icon behaves like an inline element, and that if you want to vertically align it in relation to other sibling elements, you have to use CSS to achieve the expected result.

Reviewer: ```suggestion
By default, the `FlightIcon` component has an `inline-block` display value (this can be changed using the `@isInlineBlock` argument). This means that the icon behaves like an inline element, and that if you want to vertically align it in relation to other sibling elements, you have to use CSS to achieve the expected result.
```

For example, to visually center an icon with a generic text node, you will need to use a parent `flex` container with `align-items: center`.

!!! Warning

> **Notice**

Reviewer 2: ```suggestion
**Note**
```

> **Note**

Reviewer 3: ```suggestion
**Avoid using `vertical-align: middle`**
```

Or something more descriptive

Just setting `vertical-align: middle` in the parent container doesn't necessarily achieve a vertical alignment.

> This is because, despite what one may think, the [`middle` alignment](https://developer.mozilla.org/en-US/docs/Web/CSS/vertical-align#middle) is not calculated in relation to the whole text "block" but to its “x-height”. If you want to know more, you can read [this in-depth explanation](https://www.impressivewebs.com/css-vertical-align/) of how vertical align works in CSS.

Reviewer: ```suggestion
This is because, despite what one may think, the [`middle` alignment](https://developer.mozilla.org/en-US/docs/Web/CSS/vertical-align#middle) is not calculated in relation to the whole text ”block” but to its “x-height”. If you want to know more, you can read [this in-depth explanation](https://www.impressivewebs.com/css-vertical-align/) of how vertical align works in CSS.
```
